### PR TITLE
docs: add and improve GitHub integration guide

### PR DESCRIPTION
**Pull request by tyaga001**

### CODE REVIEW

File: docs/guides/commands.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: CodeRabbit Commands
-sidebar_label: CodeRabbit Commands
+sidebar_label: All Commands
```
Comment (aravindputrevu) [inline docs/guides/commands.md:3]: Let us revamp the sidebar and these titles in the next PR. So, we can keep this specific one as is.

File: docs/platforms/azure-devops.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Integrating CodeRabbit with Azure DevOps
-description: Integrate CodeRabbit with Azure DevOps.
+title: Use CodeRabbit with Azure DevOps
```
Comment (aravindputrevu) [inline docs/platforms/azure-devops.md:2]: But actually it is integration @tyaga001 not using like database.

File: docs/platforms/github-com.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+> GitHub is the central hub for countless Open-source projects. However, for repository maintainers, a steady flow of pull requests (PRs) can quickly turn into an [overwhelming workload](https://minimaxir.com/2023/11/open-source-dead-github/).
+Those managing popular OSS repositories with numerous stars are well aware of the challenges of keeping up with code reviews, maintaining quality, and keeping the project on track.
+
+A quick search on "**has this been abandoned**" on GitHub shows a lot on what's happening across various successful repositories.
+
+![GitHub Abandoned-Search](/img/integrations/abandoned-search.png)
+
+A healthy **Maintainer Contributor** equation has a lot of pieces but Pull-requests plays a substantial role, as it is the biggest point of contact with the repo maintainers after finding an issue one could work on.
+
+We at [CodeRabbit](https://www.coderabbit.ai/), has lived through this problem and with the advent of Generative AI, code generation models, we realised we could help improve this end of developer's life.
```
Comment (aravindputrevu) [inline docs/platforms/github-com.md:19]: Let us remove all of it

File: docs/platforms/github-com.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+## How CodeRabbit Works?
+
+> CodeRabbit integrates with GitHub, automating the code review process from the moment a pull request is created. 
+
+It preprocesses the PR content, builds context, leverages Large Language Models for analysis, and then post-processes the AI response before posting the review back to GitHub. This streamlined workflow ensures thorough AI-powered code reviews without manual intervention.
+
+![How CodeRabbit Works](/img/integrations/how-codeRabbit-works.png)
+
+## Integrating with GitHub
```
Comment (aravindputrevu) [inline docs/platforms/github-com.md:31]: may be repetitive heading

**Review by aravindputrevu — CHANGES_REQUESTED** [review]

File: docs/guides/commands.md (hunk 1 above)
Comment (tyaga001) [reply docs/guides/commands.md]: reverted

File: docs/guides/delete-account.md [hunk 5]
```diff
@@ -43,8 +43,8 @@ upon selecting **Delete Account**. You can open the dropdowns for more
 information.
 
 <div align="center">
-![Delete Account](/img/guides/delete_account_light.png#gh-light-mode-only)
-![Delete Account](/img/guides/delete_account_dark.png#gh-dark-mode-only)
+![How-to-Delete-CodeRabbit-Account](/img/guides/delete_account_light.png#gh-light-mode-only)
+![How-to-Delete-CodeRabbit-Account](/img/guides/delete_account_dark.png#gh-dark-mode-only)
```
[the remark this answers is not in the document]
Comment (tyaga001) [reply docs/guides/delete-account.md:47]: @aravindputrevu, it looks like it's not so, as I didn't change the path.

File: docs/platforms/github-com.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+> **CodeRabbit** is an AI code reviewer designed to ease the challenges of code review, supporting repository maintainers and teams. It not only reviews your PRs but also provides concise summaries, identifies potential issues, and offers insights that might be missed during manual reviews.
+
+## How CodeRabbit Works?
+
+> CodeRabbit integrates with GitHub, automating the code review process from the moment a pull request is created. 
+
+It preprocesses the PR content, builds context, leverages Large Language Models for analysis, and then post-processes the AI response before posting the review back to GitHub. This streamlined workflow ensures thorough AI-powered code reviews without manual intervention.
+
+![How CodeRabbit Works](/img/integrations/how-codeRabbit-works.png)
```
[the remark this answers is not in the document]
Comment (tyaga001) [reply docs/platforms/github-com.md:18]: No, It's correct. It's the same across all the pages, and I tried static. It's failing locally.
image

File: docs/guides/delete-account.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (aravindputrevu) [reply docs/guides/delete-account.md:47]: @coderabbitai it looks like you are wrong here, images are referred in `/img` and not `/static`.

**Comment by tyaga001** [comment]
@aravindputrevu can we merge?

**Review by aravindputrevu — APPROVED** [review]

**Pull request closed without merging**